src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -58,10 +58,18 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
                 assemblyLocation = cashedAssemblyLocation;
             }
```

Comment:
While you're at it, can you please rename `cashedAssemblyLocation` to `cachedAssemblyLocation`? ;)